---
template: post
title: >-
  On the Allosteric Effect of nsSNPs and the Emerging Importance of Allosteric
  Polymorphism
date: 2019-07-12T07:20:44.756Z
journaltypes:
  - Journal Paper
journal: >-
  Journal of Molecular Biology, Vol. 431, Issue 19, 6 Sept 2019, Pg 3933-3942,
  doi: 10.1016/j.jmb.2019.07.012, PMID: 31306666
pubmed: '31306666'
url: 'https://www.sciencedirect.com/science/article/pii/S0022283619304413'
impactfactor: '5.067'
dateofacceptance: 2019-07-04T07:20:44.768Z
description: >-
  The molecular mechanisms of pathological non-synonymous single-nucleotide
  polymorphisms are still the object of intensive research. To this end, we
  explore here whether non-synonymous single-nucleotide polymorphisms can work
  via allosteric mechanisms. Using structure-based statistical mechanical model
  of allostery and analyzing energetics of the effects of mutations in a set of
  27 proteins with at least 50 pathological SNPs in each molecule, we found
  that, indeed, some SNPs can work allosterically. 
uploadfile: /media/uploads/on-the-allosteric.pdf
tags:
  - Tee WV
  - Guarnera E
  - Berezovsky IN
categories:
  - Physics and Evolution of Biological Macromolecules
  - Biomolecular Modelling and Design Division
---
The molecular mechanisms of pathological non-synonymous single-nucleotide polymorphisms are still the object of intensive research. To this end, we explore here whether non-synonymous single-nucleotide polymorphisms can work via allosteric mechanisms. Using structure-based statistical mechanical model of allostery and analyzing energetics of the effects of mutations in a set of 27 proteins with at least 50 pathological SNPs in each molecule, we found that, indeed, some SNPs can work allosterically. We illustrate the molecular basis of disease phenotypes caused by allosteric SNPs with the case studies of human galactose 1-phosphate uridyltransferase (GALT) and glucose-6-phosphate dehydrogenase (G6PD). We also found that mutations of a number of other residues in the protein may cause modulation comparable to those observed for known pathological SNPs. In order to explain this, we propose a notion of allosteric polymorphism, which implies the presence of a number of critical positions in the protein sequence, whose mutations can allosterically disrupt the protein function and result in a disease phenotype. We conclude that the emerging importance of allosteric polymorphism calls for the development of computational framework for analyzing the allosteric effects of mutations and their role in the modulation of protein activity.
