---
template: post
title: Exploring chromatin hierarchical organization via Markov State Modelling
date: 2018-12-31T03:52:54.713Z
journaltypes: Journal Paper
journal: >-
  PLoS Computational Biology, 2018 Dec 31;14(12), doi:
  10.1371/journal.pcbi.100668
pubmed: '30596637 '
url: 'https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006686'
impactfactor: '3.955'
dateofacceptance: 2018-11-27T03:52:54.723Z
description: >-
  We propose a new computational method for exploring chromatin structural
  organization based on Markov State Modelling of Hi-C data represented as an
  interaction network between genomic loci. A Markov process describes the
  random walk of a traveling probe in the corresponding energy landscape,
  mimicking the motion of a biomolecule involved in chromatin function. By
  studying the metastability of the associated Markov State Model upon
  annealing, the hierarchical structure of individual chromosomes is observed,
  and corresponding set of structural partitions is identified at each level of
  hierarchy. 
uploadfile: /media/uploads/17.pdf
tags:
  - Tan ZW
  - Guarnera E
  - Berezovsky IN
categories:
  - Physics and Evolution of Biological Macromolecules
  - Biomolecular Modelling and Design Division
---
<!--StartFragment-->

We propose a new computational method for exploring chromatin structural organization based on Markov State Modelling of Hi-C data represented as an interaction network between genomic loci. A Markov process describes the random walk of a traveling probe in the corresponding energy landscape, mimicking the motion of a biomolecule involved in chromatin function. By studying the metastability of the associated Markov State Model upon annealing, the hierarchical structure of individual chromosomes is observed, and corresponding set of structural partitions is identified at each level of hierarchy. Then, the notion of effective interaction between partitions is derived, delineating the overall topology and architecture of chromosomes. Mapping epigenetic data on the graphs of intra-chromosomal effective interactions helps in understanding how chromosome organization facilitates its function. A sketch of whole-genome interactions obtained from the analysis of 539 partitions from all 23 chromosomes, complemented by distributions of gene expression regulators and epigenetic factors, sheds light on the structure-function relationships in chromatin, delineating chromosomal territories, as well as structural partitions analogous to topologically associating domains and active / passive epigenomic compartments. In addition to the overall genome architecture shown by effective interactions, the affinity between partitions of different chromosomes was analyzed as an indicator of the degree of association between partitions in functionally relevant genomic interactions. The overall static picture of whole-genome interactions obtained with the method presented in this work provides a foundation for chromatin structural reconstruction, for the modelling of chromatin dynamics, and for exploring the regulation of genome function. The algorithms used in this study are implemented in a freely available Python package ChromaWalker (<https://bitbucket.org/ZhenWahTan/chromawalker>).

<!--EndFragment-->
