---
template: post
title: Functional Classification of Super-Large Families of Enzymes Based on
  Substrate Binding Pocket Residues for Biocatalysis and Enzyme Engineering
  Applications
date: 2021-08-02T07:36:16.834Z
journaltypes: Journal Paper
journal: "Frontiers in Bioengineering and Biotechnology, 2 Aug 2021, doi:
  10.3389/fbioe.2021.701120"
pubmed: "34409021"
url: https://www.frontiersin.org/articles/10.3389/fbioe.2021.701120/full
impactfactor: "5.89"
dateofacceptance: 2021-07-12T07:36:16.848Z
description: "Large enzyme families such as the groups of zinc-dependent alcohol
  dehydrogenases (ADHs), long chain alcohol oxidases (AOxs) or amine
  dehydrogenases (AmDHs) with, sometimes, more than one million sequences in the
  non-redundant protein database and hundreds of experimentally characterized
  enzymes are excellent cases for protein engineering efforts aimed at refining
  and modifying substrate specificity. "
tags:
  - Sirota FL
  - Maurer-Stroh S
  - Li Z
  - Eisenhaber F
  - Eisenhaber B
categories:
  - "Biomolecular Function Discovery Division "
  - Gene Function Prediction
  - Protein Sequence Analysis
---
Large enzyme families such as the groups of zinc-dependent alcohol dehydrogenases (ADHs), long chain alcohol oxidases (AOxs) or amine dehydrogenases (AmDHs) with, sometimes, more than one million sequences in the non-redundant protein database and hundreds of experimentally characterized enzymes are excellent cases for protein engineering efforts aimed at refining and modifying substrate specificity. Yet, the backside of this wealth of information is that it becomes technically difficult to rationally select optimal sequence targets as well as sequence positions for mutagenesis studies. In all three cases, we approach the problem by starting with a group of experimentally well studied family members (including those with available 3D structures) and creating a structure-guided multiple sequence alignment and a modified phylogenetic tree (aka binding site tree) based just on a selection of potential substrate binding residue positions derived from experimental information (not from the full-length sequence alignment). Hereupon, the remaining, mostly uncharacterized enzyme sequences can be mapped; as a trend, sequence grouping in the tree branches follows substrate specificity. We show that this information can be used in the target selection for protein engineering work to narrow down to single suitable sequences and just a few relevant candidate positions for directed evolution towards activity for desired organic compound substrates. We also demonstrate how to find the closest thermophile example in the dataset if the engineering is aimed at achieving most robust enzymes.