---
template: post
title: "To kill or to be killed: pangenome analysis of Escherichia coli strains
  reveals a tailocin specific for pandemic ST131 "
date: 2022-06-16T06:10:00.000Z
journaltypes: Journal Paper
journal: "BMC Biology 2022 Jun 16;20(1):146. doi: 10.1186/s12915-022-01347-7"
pubmed: "35710371"
url: https://bmcbiol.biomedcentral.com/articles/10.1186/s12915-022-01347-7
impactfactor: "7.4"
dateofacceptance: 2022-06-08T06:10:00.000Z
description: Escherichia coli is one of the most well-known commensal
  Gram-negative bacteria, which is commonly associated with the gut microbiome.
  Since first identified in 1844, it has been widely studied as a model organism
  in the laboratory. However, recent findings have shown not only the
  versatility of E. coli living in different ecological niches but also the
  diversity of its genotypes including strains with pathogenicity for animals
  and human [1, 2].
uploadfile: /media/uploads/1319_to-kill-or-to-be-killed.pdf
tags:
  - Tantoso E
  - Eisenhaber B
  - Kirsch M
  - Shitov V
  - Zhao Z
  - Eisenhaber F
categories:
  - "Biomolecular Function Discovery Division "
  - Gene Function Prediction
---
<!--StartFragment-->

### Background

*Escherichia coli* (*E. coli*) has been one of the most studied model organisms in the history of life sciences. Initially thought just to be commensal bacteria, *E. coli* has shown wide phenotypic diversity including pathogenic isolates with great relevance to public health. Though pangenome analysis has been attempted several times, there is no systematic functional characterization of the *E. coli* subgroups according to the gene profile.

### Results

Systematically scanning for optimal parametrization, we have built the *E. coli* pangenome from 1324 complete genomes. The pangenome size is estimated to be \~25,000 gene families (GFs). Whereas the core genome diminishes as more genomes are added, the softcore genome (â‰¥95% of strains) is stable with \~3000 GFs regardless of the total number of genomes. Apparently, the softcore genome (with a 92% or 95% generation threshold) can define the genome of a bacterial species listing the critically relevant, evolutionarily most conserved or important classes of GFs. Unsupervised clustering of common *E. coli* sequence types using the presence/absence GF matrix reveals distinct characteristics of *E. coli* phylogroups B1, B2, and E. We highlight the bi-lineage nature of B1, the variation of the secretion and of the iron acquisition systems in ST11 (E), and the incorporation of a highly conserved prophage into the genome of ST131 (B2). The tail structure of the prophage is evolutionarily related to R2-pyocin (a tailocin) from *Pseudomonas aeruginosa* PAO1. We hypothesize that this molecular machinery is highly likely to play an important role in protecting its own colonies; thus, contributing towards the rapid rise of pandemic *E. coli* ST131.

### Conclusions

This study has explored the optimized pangenome development in *E. coli*. We provide complete GF lists and the pangenome matrix as supplementary data for further studies. We identified biological characteristics of different *E. coli* subtypes, specifically for phylogroups B1, B2, and E. We found an operon-like genome region coding for a tailocin specific for ST131 strains. The latter is a potential killer weapon providing pandemic *E. coli* ST131 with an advantage in inter-bacterial competition and, suggestively, explains their dominance as human pathogen among *E. coli* strains.

<!--EndFragment-->