---
template: post
title: About the dark corners in the gene function space of Escherichia coli
  remaining without illumination by scientific literature
date: 2023-02-28T13:17:00.000Z
journaltypes: Journal Paper
journal: "Biology Direct, 2023 Feb 28;18(1):7. doi: 10.1186/s13062-023-00362-0"
pubmed: "36855185"
url: https://biologydirect.biomedcentral.com/articles/10.1186/s13062-023-00362-0
impactfactor: "7.173"
dateofacceptance: 2023-02-21T13:17:00.000Z
description: Although Escherichia coli (E. coli) is the most studied prokaryote
  organism in the history of life sciences, many molecular mechanisms and gene
  functions encoded in its genome remain to be discovered. This work aims at
  quantifying the illumination of the E. coli gene function space by the
  scientific literature and how close we are towards the goal of a complete list
  of E. coli gene functions.
uploadfile: /media/uploads/1356_about-the-dark-corners-in-the-gene-function-space-of-escherichia-coli.pdf
tags:
  - Tantoso E
  - Eisenhaber B
  - Sinha S
  - Jensen LJ
  - Eisenhaber F
categories:
  - "Biomolecular Function Discovery Division "
  - Gene Function Prediction
---
<!--StartFragment-->

### Background

Although *Escherichia coli* (*E. coli*) is the most studied prokaryote organism in the history of life sciences, many molecular mechanisms and gene functions encoded in its genome remain to be discovered. This work aims at quantifying the illumination of the *E. coli* gene function space by the scientific literature and how close we are towards the goal of a complete list of *E. coli* gene functions.

### Results

The scientific literature about *E. coli* protein-coding genes has been mapped onto the genome via the mentioning of names for genomic regions in scientific articles both for the case of the strain K-12 MG1655 as well as for the 95%-threshold softcore genome of 1324 *E. coli* strains with known complete genome. The article match was quantified with the ratio of a given gene name’s occurrence to the mentioning of any gene names in the paper. The various genome regions have an extremely uneven literature coverage. A group of elite genes with ≥ 100 full publication equivalents (FPEs, FPE = 1 is an idealized publication devoted to just a single gene) attracts the lion share of the papers. For K-12, \~ 65% of the literature covers just 342 elite genes; for the softcore genome, \~ 68% of the FPEs is about only 342 elite gene families (GFs). We also find that most genes/GFs have at least one mentioning in a dedicated scientific article (with the exception of at least 137 protein-coding transcripts for K-12 and 26 GFs from the softcore genome). Whereas the literature growth rates were highest for uncharacterized or understudied genes until 2005–2010 compared with other groups of genes, they became negative thereafter. At the same time, literature for anyhow well-studied genes started to grow explosively with threshold T10 (≥ 10 FPEs). Typically, a body of \~ 20 actual articles generated over \~ 15 years of research effort was necessary to reach T10. Lineage-specific co-occurrence analysis of genes belonging to the accessory genome of *E. coli* together with genomic co-localization and sequence-analytic exploration hints previously completely uncharacterized genes *yahV* and *yddL* being associated with osmotic stress response/motility mechanisms.

### Conclusion

If the numbers of scientific articles about uncharacterized and understudied genes remain at least at present levels, full gene function lists for the strain K-12 MG1655 and the *E. coli* softcore genome are in reach within the next 25–30 years. Once the literature body for a gene crosses 10 FPEs, most of the critical fundamental research risk appears overcome and steady incremental research becomes possible.

<!--EndFragment-->