---
template: post
title: A novel 3-miRNA network regulates tumour  progression in oral squamous
  cell carcinoma
date: 2023-06-14T14:05:00.000Z
journaltypes: Journal Paper
journal: "Biomarker Research 2023 Jun 14;11(1):64. doi:
  10.1186/s40364-023-00505-5. PMID: 37316916"
pubmed: "37175867"
impactfactor: "6.208"
dateofacceptance: 2023-03-19T14:05:00.000Z
description: This study identified potential salivary exosome-derived miRNA
  biomarkers and crucial miRNA-mRNA networks/underlying mechanisms responsible
  for OSCC progression.
uploadfile: /media/uploads/1365_a-novel-3-mirna-network-regulates.pdf
tags:
  - Patel A
  - Patel P
  - Mandlik D
  - Patel K
  - Malaviya P
  - Johar K
  - Swamy KBS
  - Patel S
  - Tanavde V
categories:
  - Gene Function Prediction
---
<!--StartFragment-->

### Background

Late diagnosis is one of the major confounders in oral squamous cell carcinoma (OSCC). Despite recent advances in molecular diagnostics, no disease-specific biomarkers are clinically available for early risk prediction of OSCC. Therefore, it is important to identify robust biomarkers that are detectable using non-invasive liquid biopsy techniques to facilitate the early diagnosis of oral cancer. This study identified potential salivary exosome-derived miRNA biomarkers and crucial miRNA-mRNA networks/underlying mechanisms responsible for OSCC progression.

### Methods

Small RNASeq (n = 23) was performed in order to identify potential miRNA biomarkers in both tissue and salivary exosomes derived from OSCC patients. Further, integrated analysis of The Cancer Genome Atlas (TCGA) datasets (n = 114), qPCR validation on larger patient cohorts (n = 70) and statistical analysis with various clinicopathological parameters was conducted to assess the effectiveness of the identified miRNA signature. miRNA-mRNA networks and pathway analysis was conducted by integrating the transcriptome sequencing and TCGA data. The OECM-1 cell line was transfected with the identified miRNA signature in order to observe its effect on various functional mechanisms such as cell proliferation, cell cycle, apoptosis, invasive as well as migratory potential and the downstream signaling pathways regulated by these miRNA-mRNA networks.

### Results

Small RNASeq and TCGA data identified 12 differentially expressed miRNAs in OSCC patients compared to controls. On validating these findings in a larger cohort of patients, miR-140-5p, miR-143-5p, and miR-145-5p were found to be significantly downregulated. This 3-miRNA signature demonstrated higher efficacy in predicting disease progression and clinically correlated with poor prognosis (p < 0.05). Transcriptome, TCGA, and miRNA-mRNA network analysis identified HIF1a, CDH1, CD44, EGFR, and CCND1 as hub genes regulated by the miRNA signature. Further, transfection-mediated upregulation of the 3-miRNA signature significantly decreased cell proliferation, induced apoptosis, resulted in G2/M phase cell cycle arrest and reduced the invasive and migratory potential by reversing the EMT process in the OECM-1 cell line.

### Conclusions

Thus, this study identifies a 3-miRNA signature that can be utilized as a potential biomarker for predicting disease progression of OSCC and uncovers the underlying mechanisms responsible for converting a normal epithelial cell into a malignant phenotype.

<!--EndFragment-->