---
template: post
title: "Outbreak.info genomic reports: scalable and dynamic surveillance of
  SARS-CoV-2 variants and mutations"
date: 2023-02-23T09:45:00.000Z
journaltypes: Journal Paper
journal: "Nature Methods 2023 Apr;20(4):512-522. doi: 10.1038/s41592-023-01769-3"
pubmed: "36823332"
url: https://www.nature.com/articles/s41592-023-01769-3#citeas
impactfactor: "48.0"
dateofacceptance: 2023-01-17T09:45:00.000Z
description: "In response to the emergence of SARS-CoV-2 variants of concern,
  the global scientific community, through unprecedented effort, has sequenced
  and shared over 11 million genomes through GISAID, as of May 2022. "
uploadfile: /media/uploads/1401_outbreak-info-genomic-reports.pdf
tags:
  - Gangavarapu K
  - Latif AA
  - Mullen JL
  - Alkuzweny M
  - Hufbauer E
  - Tsueng G
  - Haag E
  - Zeller M
  - Aceves CM
  - Zaiets K
  - Cano M
  - Zhou X
  - Qian Z
  - Sattler R
  - Matteson NL
  - Levy JI
  - Lee RTC
  - Freitas L
  - Maurer-Stroh S; GISAID Core and Curation Team; Suchard MA
  - Wu C
  - Su AI
  - Andersen KG
  - Hughes LD
categories:
  - Protein Sequence Analysis
  - "Biomolecular Function Discovery Division "
---
<!--StartFragment-->

In response to the emergence of SARS-CoV-2 variants of concern, the global scientific community, through unprecedented effort, has sequenced and shared over 11 million genomes through GISAID, as of May 2022. This extraordinarily high sampling rate provides a unique opportunity to track the evolution of the virus in near real-time. Here, we present [outbreak.info](https://outbreak.info/), a platform that currently tracks over 40 million combinations of Pango lineages and individual mutations, across over 7,000 locations, to provide insights for researchers, public health officials and the general public. We describe the interpretable visualizations available in our web application, the pipelines that enable the scalable ingestion of heterogeneous sources of SARS-CoV-2 variant data and the server infrastructure that enables widespread data dissemination via a high-performance API that can be accessed using an R package. We show how [outbreak.info](https://outbreak.info/) can be used for genomic surveillance and as a hypothesis-generation tool to understand the ongoing pandemic at varying geographic and temporal scales.

<!--EndFragment-->