---
template: post
title: Sub genomic analysis of SARS-CoV-2 using short read amplicon-based sequencing
date: 2023-02-24T08:34:00.000Z
journaltypes: Journal Paper
journal: "Frontiers in Genetics, Volume 14 - 2023, doi: 10.3389/fgene.2023.1086865"
pubmed: "36911398"
url: https://www.frontiersin.org/journals/genetics/articles/10.3389/fgene.2023.1086865/full
impactfactor: "3.7"
dateofacceptance: 2023-01-25T08:34:00.000Z
description: The novel coronavirus disease 2019 (COVID-19) pandemic poses a
  serious public health risk. In this report, we present a modified sequencing
  workflow using short tiling (280bp) amplicons library preparation method
  paired with Illumina’s iSeq100 desktop sequencer.
uploadfile: /media/uploads/1413_towards-a-rapid-turnaround-low-depth.pdf
tags:
  - Koh WLC
  - Poh SE
  - Lee CK
  - Chan THM
  - Yan G
  - Kong KW
  - Lau L
  - Lee WYT
  - Cheng C
  - Hoon S
  - Seow Y
categories:
  - Biomolecular Sequence To Function Division
  - Molecular Innovations And AI Integrations
---
<!--StartFragment-->

The novel coronavirus disease 2019 (COVID-19) pandemic poses a serious public health risk. In this report, we present a modified sequencing workflow using short tiling (280bp) amplicons library preparation method paired with Illumina’s iSeq100 desktop sequencer. We demonstrated the utility of our workflow in identifying gapped reads that capture characteristics of subgenomic RNA junctions within our patient cohort. These analytical and library preparation approaches allow a versatile, small footprint and decentralized deployment that can facilitate comprehensive genetics characterizations during outbreaks. Based on the sequencing data, Taqman assays were designed to accurately capture the quantity of subgenomic ORF5 and ORF7a RNA from patient samples and demonstrated utility in tracking subgenomic titres in patient samples when combined with a standard COVID-19 qRT-PCR assay.

<!--EndFragment-->