---
template: post
title: Deciphering spatial domains from spatial multi-omics with SpatialGlue
date: 2024-06-21T07:07:29.486Z
journaltypes: Journal Paper
journal: "Nature Methods 2024, doi: 10.1038/s41592-024-02316-4"
pubmed: "38907114"
url: https://www.nature.com/articles/s41592-024-02316-4
impactfactor: "45.6"
dateofacceptance: 2024-08-29T07:07:29.502Z
description: Advances in spatial omics technologies now allow multiple types of
  data to be acquired from the same tissue slice. To realize the full potential
  of such data, we need spatially informed methods for data integration.
uploadfile: /media/uploads/1458_deciphering-spatial-domains-from.pdf
tags:
  - Long Y
  - Ang KS
  - Sethi R
  - Liao S
  - Heng Y
  - van Olst L
  - Ye S
  - Zhong C
  - Xu H
  - Zhang D
  - Kwok I
  - Husna N
  - Jian M
  - Ng LG
  - Chen A
  - Gascoigne NRJ
  - Gate D
  - Fan R
  - Xu X
  - Chen J
categories:
  - Cellular Image Informatics Division
---
<!--StartFragment-->

Advances in spatial omics technologies now allow multiple types of data to be acquired from the same tissue slice. To realize the full potential of such data, we need spatially informed methods for data integration. Here, we introduce SpatialGlue, a graph neural network model with a dual-attention mechanism that deciphers spatial domains by intra-omics integration of spatial location and omics measurement followed by cross-omics integration. We demonstrated SpatialGlue on data acquired from different tissue types using different technologies, including spatial epigenome–transcriptome and transcriptome–proteome modalities. Compared to other methods, SpatialGlue captured more anatomical details and more accurately resolved spatial domains such as the cortex layers of the brain. Our method also identified cell types like spleen macrophage subsets located at three different zones that were not available in the original data annotations. SpatialGlue scales well with data size and can be used to integrate three modalities. Our spatial multi-omics analysis tool combines the information from complementary omics modalities to obtain a holistic view of cellular and tissue properties.

<!--EndFragment-->