---
template: post
title: "Identification and semi-quantification of protein allergens in complex
  mixtures using proteomic and AllerCatPro 2.0 bioinformatic analyses: a
  proof-of-concept investigation"
date: 2024-01-30T17:07:54.671Z
journaltypes: Journal Paper
journal: "Journal of Immunotoxicology, Vol. 21, 2024, Issue 1, doi:
  10.1080/1547691X.2024.2305452"
pubmed: "38291955"
url: https://www.tandfonline.com/doi/full/10.1080/1547691X.2024.2305452
impactfactor: "3.3"
dateofacceptance: 2024-01-08T17:07:54.687Z
description: "Keywords: IgE; Protein allergenicity; botanicals; natural
  substances; risk assessment. "
uploadfile: /media/uploads/1451_identification-and-semi-quantification-of-protein.pdf
tags:
  - Krutz NL
  - Kimber I
  - Winget J
  - Nguyen MN
  - Limviphuvadh V
  - Maurer-Stroh S
  - Mahony C
  - Gerberick GF
categories:
  - Atomistic Simulations and Design in Biology
  - Biomolecular Structure To Mechanism Division
  - Protein Sequence Analysis
  - Biomolecular Sequence To Function Division
---
<!--StartFragment-->

The demand for botanicals and natural substances in consumer products has increased in recent years. These substances usually contain proteins and these, in turn, can pose a risk for immunoglobulin E (IgE)-mediated sensitization and allergy. However, no method has yet been accepted or validated for assessment of potential allergenic hazards in such materials. In the studies here, a dual proteomic-bioinformatic approach is proposed to evaluate holistically allergenic hazards in complex mixtures of plants, insects, or animal proteins. Twelve commercial preparations of source materials (plant products, dust mite extract, and preparations of animal dander) known to contain allergenic proteins were analyzed by label-free proteomic analyses to identify and semi-quantify proteins. These were then evaluated by bioinformatics using AllerCatPro 2.0 (https://allercatpro.bii.a-star.edu.sg/) to predict no, weak, or strong evidence for allergenicity and similarity to source-specific allergens. In total, 4,586 protein sequences were identified in the 12 source materials combined. Of these, 1,665 sequences were predicted with weak or strong evidence for allergenic potential. This first-tier approach provided top-level information about the occurrence and abundance of proteins and potential allergens. With regards to source-specific allergens, 129 allergens were identified. The sum of the relative abundance of these allergens ranged from 0.8% (lamb's quarters) to 63% (olive pollen). It is proposed here that this dual proteomic-bioinformatic approach has the potential to provide detailed information on the presence and relative abundance of allergens, and can play an important role in identifying potential allergenic hazards in complex protein mixtures for the purposes of safety assessments.

<!--EndFragment-->