---
template: post
title: Towards a Rapid-Turnaround Low-Depth Unbiased Metagenomics Sequencing
  Workflow on the Illumina Platforms
date: 2023-04-25T08:28:00.000Z
journaltypes: Journal Paper
journal: "Bioengineering 2023, 10(5), 520, doi: 10.3390/bioengineering10050520"
pubmed: "37237590"
url: https://www.mdpi.com/2306-5354/10/5/520
impactfactor: "4.6"
dateofacceptance: 2023-04-21T08:29:00.000Z
description: Unbiased metagenomic sequencing is conceptually well-suited for
  first-line diagnosis as all known and unknown infectious entities can be
  detected, but costs, turnaround time and human background reads in complex
  biofluids, such as plasma, hinder widespread deployment.
uploadfile: /media/uploads/1413_towards-a-rapid-turnaround-low-depth.pdf
tags:
  - Koh WLC
  - Poh SE
  - Lee CK
  - Chan THM
  - Yan G
  - Kong KW
  - Lau L
  - Lee WYT
  - Cheng C
  - Hoon S
  - Seow Y
categories:
  - Biomolecular Sequence To Function Division
  - Molecular Innovations And AI Integrations
---
<!--StartFragment-->

Unbiased metagenomic sequencing is conceptually well-suited for first-line diagnosis as all known and unknown infectious entities can be detected, but costs, turnaround time and human background reads in complex biofluids, such as plasma, hinder widespread deployment. Separate preparations of DNA and RNA also increases costs. In this study, we developed a rapid unbiased metagenomics next-generation sequencing (mNGS) workflow with a human background depletion method (HostEL) and a combined DNA/RNA library preparation kit (AmpRE) to address this issue. We enriched and detected bacterial and fungal standards spiked in plasma at physiological levels with low-depth sequencing (<1 million reads) for analytical validation. Clinical validation also showed 93% of plasma samples agreed with the clinical diagnostic test results when the diagnostic qPCR had a Ct < 33. The effect of different sequencing times was evaluated with the 19 h iSeq 100 paired end run, a more clinically palatable simulated iSeq 100 truncated run and the rapid 7 h MiniSeq platform. Our results demonstrate the ability to detect both DNA and RNA pathogens with low-depth sequencing and that iSeq 100 and MiniSeq platforms are compatible with unbiased low-depth metagenomics identification with the HostEL and AmpRE workflow.

<!--EndFragment-->